### ISSUE TITLE
[Android] Worker doesn't work in 50.2.2.5

### ISSUE THREAD

**Issue by Borman2000** [airsdk/Adobe-Runtime-Support#2661]
## Problem Description

Our old worker works fine in 20.2.1.1, but doesn't work in 50.2.2.5 - worker constructor never called (after WorkerDomain.current.createWorker(swfBytes))

Is there any changes in workers flow or configuration that I missed?
Looked at 50.2.2.* release notes and haven't find anything related.

## Known Workarounds

none

**Comment by ajwfrost** [comment]
There's one known issue in 50.2.2.5 which is related to #2385, where a worker could fail to start if there are ANEs involved (a side-effect of us trying to ensure that ANE library definitions are loaded into worker contexts). So if you have ANEs in your project, can you please try disabling them (or replacing them with stubbed SWC files) and see if that helps?

Otherwise, if you can reproduce this with a really basic example, please let us know and we can look at reproducing it with the set-up you have (xml descriptor details etc the same). Our simple worker test case is currently fine though so if it's not the ANE thing, I'm not sure what it could be.

thanks

**Comment by Borman2000** [comment]
> There's one known issue in 50.2.2.5 which is related to #2385, where a worker could fail to start if there are ANEs involved

Yes, I saw this ticket and read about ANE. We have a lot of them in project but no one is used in the worker.
At least directly... I'll double check in Monday and let you know.
Thanks for putting this idea into my head. :)

**Comment by Borman2000** [comment]
Figured out that having at least one ANE in manifest's <extensions> section is enough to break worker.
You don't even need to use this ANE in you code (neither main code nor worker code). Just define it in manifest.
Checked with very basic worker example from adobe's docs.

**Comment by bishpatrik** [comment]
Just noticed 50.2.2.6 was available, so I have upgraded and it appears to be working again.
